src/components/Card/CardEdit.tsx

```diff
@@ -19,13 +19,17 @@ const CardEdit: React.FC<ICardEdit> = ({ handleSwitchEdit, type, card }) => {
   const handleSave = async () => {
     await updateCard(card, textToEdit);
     handleSwitchEdit();
-    await getCards(setFlashCards);
+    if (getAllCards) {
+      await getAllCards();
```

Comment:
**nitpick:** You'd have to add these `if` all the time adding complexity to your codebase - let's fix it, so you don't have to do this.